chainio/clients/eigenpod/reader.go

```diff
@@ -0,0 +1,74 @@
+package eigenpod
+
+import (
+	"github.com/Layr-Labs/eigensdk-go/chainio/clients/eigenpod/bindings"
+	"github.com/Layr-Labs/eigensdk-go/chainio/clients/eth"
+	"github.com/Layr-Labs/eigensdk-go/logging"
+	"github.com/ethereum/go-ethereum/common"
+)
+
+type ChainReader struct {
+	logger    logging.Logger
+	ethClient eth.HttpBackend
+	*bindings.IEigenPodCaller
+}
+
+type ManagerChainReader struct {
+	logger    logging.Logger
+	ethClient eth.HttpBackend
+	*bindings.IEigenPodManagerCaller
+}
```

Comment:
why not merge them in a single struct like we do everything else?
If there's a good reason then add as comment above the structs

Reply:
yea there is. most people who want to manage an eigenpod only need the first one. Eigenpod manager is mostly for creating a new eigenpod. I will add that comment.

Reply:
c68b91180062786b36dc2566fb7dc393ac673d89